tools/nni_cmd/config_utils.py

```diff
@@ -9,8 +9,8 @@
 
 class Config:
     '''a util class to load and save config'''
-    def __init__(self, file_path):
-        config_path = os.path.join(NNICTL_HOME_DIR, str(file_path))
+    def __init__(self, file_path, home_dir = NNICTL_HOME_DIR):
```

Comment:
better to remove spaces around `=` for default values.

Reply:
updated, seems pylint didn't check this